### PR TITLE
Adding checkstyle for test files

### PR DESCRIPTION
**Pull request by sagdutt**

### CODE REVIEW

File: src/test/java/com/bhoruka/bloodbank/dao/integration/DynamoDbLocalTest.java [hunk 1]
```diff
@@ -25,15 +25,15 @@ public void test() {
             listTables(dynamodb.listTables(), "DynamoDB Embedded");
         } finally {
             // Shutdown the thread pools in DynamoDB Local / Embedded
-            if(dynamodb != null) {
+            if (dynamodb != null) {
```
Comment (anubratagunner) [inline src/test/java/com/bhoruka/bloodbank/dao/integration/DynamoDbLocalTest.java]: Is putting a space before/after braces necessary?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+language: java
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```
Comment (anubratagunner) [inline .travis.yml]: Shouldn't there be a newline?

Comment (sagdutt) [reply .travis.yml]: doesn't really matter

File: src/test/java/com/bhoruka/bloodbank/dao/integration/DynamoDbLocalTest.java (hunk 1 above)
Comment (sagdutt) [reply src/test/java/com/bhoruka/bloodbank/dao/integration/DynamoDbLocalTest.java]: yes, it's part of our checkstyle which is based on google_checks. It makes the code more readable. More info here: https://checkstyle.sourceforge.io/google_style.html

**Pull request merged**